azext_iot/operations/hub.py

```diff
@@ -1125,6 +1126,7 @@ def iot_hub_monitor_events(cmd, hub_name=None, device_id=None, consumer_group='$
     timeout = (timeout * 1000)
 
     config = cmd.cli_ctx.config
+    output = cmd.cli_ctx.invocation.data.get("output", None) #get_output_format(cmd.cli_ctx)
```

Comment:
Ignore this for now.  I'm working with the CLI team to figure out the best way to get a global parameter from within an extension.